api/workflow/definition/processes.js

```diff
@@ -44,14 +44,19 @@ export const showMilestone = {
 //     version: '1.0',
 //     title: 'Test Workflow',
 //     description: 'This is a test process',
-//     position: 0,
 //     steps_attributes: [
 //       {
-//         title: 'Step 1', description: 'This is step 1', kind: 'decision', completion_type: 'each_person',
+//         title: 'Step 1', description: 'This is step 1', kind: 'decision', completion_type: 'each_person', min_worktime: 5, max_worktime: 10,
 //         decision_options_attributes: [{description: "option 1"}, {description: "option 2"}],
 //         documents_attributes: [{title: "document title", link: "www.example.com"}]
 //       },
 //       { title: 'Step 2', description: 'This is step 2', kind: 'default', completion_type: 'one_per_group' }
+//     ],
+//     selected_processes_attributes: [
+//       { workflow_id: 35, position: 0}
+//     ],
+//     workable_dependencies_attributes: [
+//       { workflow_id: 35, prerequisite_workable_type: "Workflow::Definition::Process", prerequisite_workable_id: 6},
```

Comment:
This is to create a `prerequisite`. The prerequisite milestone id would be the `prerequisite_workable_id`